src/components/Badge.js

```diff
@@ -19,8 +19,18 @@ const BadgeDiv = styled.span`
   margin: 5px;
 `;
 
-export default function Badge({ active, children }) {
-  return <BadgeDiv active={active}>{children}</BadgeDiv>;
+export default function Badge({ children, onClick, active }) {
+  return (
+    <BadgeDiv
+      onClick={() => {
+        onClick();
+        console.log(active);
```

Comment:
To avoid having logs which were only used during development, we try to remove logs in the final version.
It is enough here to write `onClick={onClick} `